oxidauth-kernel/src/auth/register.rs

```diff
@@ -31,10 +29,18 @@ pub struct RegisterResponse {
     pub refresh_token: Uuid,
 }
 
-impl From<&RegisterParams> for FindAuthorityByStrategy {
+// impl From<&RegisterParams> for FindAuthorityByStrategy {
+//     fn from(value: &RegisterParams) -> Self {
+//         Self {
+//             strategy: value.strategy,
+//         }
+//     }
+// }
```

Comment:
We should either uncomment this or delete it :).

```suggestion
// impl From<&RegisterParams> for FindAuthorityByStrategy {
//     fn from(value: &RegisterParams) -> Self {
//         Self {
//             strategy: value.strategy,
//         }
//     }
// }
```